CHANGELOG.md

```diff
@@ -9,15 +9,19 @@ Then it was renamed Kiwix-html5 (and uses ZIM files), and then again was renamed
 Released on *TODO*
 
 * FEATURE: Provided a workaround to enable Service Worker mode in Firefox extensions (via a PWA)
+* NEW: App now defaults to Service Worker mode if accessed as a PWA
```

Comment:
It would be better to give a more specific explanation here.
With the line above, one could conclude that we now default to SW mode in Firefox extensions (which is wrong)

```suggestion
* NEW: App now defaults to Service Worker mode if accessed directly as a PWA (does not apply to browser extensions)
```

Reply:
Thanks, I committed that change.